lib/ruby_lexer.rb

```diff
@@ -22,6 +22,7 @@ class RubyLexer
   STR_FUNC_QWORDS = 0x08
   STR_FUNC_SYMBOL = 0x10
   STR_FUNC_INDENT = 0x20 # <<-HEREDOC
+  STR_FUNC_ICNTNT = 0x40 # <<~HEREDOC
```

Comment:
Btw got it from here: https://ruby-doc.org/core-2.3.0/doc/syntax/literals_rdoc.html#label-Here+Documents (search for "indented content" in that section).